Update latest spec to 6.10.0

nicholas-tao opened the pull request:
NEW FEATURES:

* Added new fields to `Restore` model: `BackupEndTime`, `ClientErrorCode`, `ClientErrorMessage`, `CompletedAt`, `CrdbJobId`, `SourceClusterName`, `DestinationClusterName`, `Objects`, `RestoreOpts`
* Added new audit log actions: `START_FAULT_TOLERANCE_DEMO`, `END_FAULT_TOLERANCE_DEMO`

linhcrl commented on CHANGELOG.md:
```diff
@@ -1,3 +1,10 @@
+
+NEW FEATURES:
+
+* Add new fields to `Restore` model: `BackupEndTime`, `ClientErrorCode`, `ClientErrorMessage`, `CompletedAt`, `CrdbJobId`, `SourceClusterName`, `DestinationClusterName`, `Objects`, `Opts`
```
One more thing: we usually mark fields that we expect to be populated as required in the api.proto like [this](https://github.com/cockroachlabs/managed-service/blob/0dd85fa14c509c654e759a4025a732630cdf4464/console/apipb/api.proto#L4788). I would expect some of the newly added fields to be there. Once you make this change, the generated sdk client will change because some fields will no longer be pointers. Can you please update that as well when you rename `Opts`?

nicholas-tao committed:
Update latest spec to 6.10.0

NEW FEATURES:

* Added new fields to `Restore` model: `BackupEndTime`,
`ClientErrorCode`, `ClientErrorMessage`, `CompletedAt`, `CrdbJobId`,
`SourceClusterName`, `DestinationClusterName`, `Objects`, `RestoreOpts`
* Added new audit log actions: `START_FAULT_TOLERANCE_DEMO`,
`END_FAULT_TOLERANCE_DEMO`

linhcrl reviewed (approved):
LGTM! thank you!!
